### DISCUSSION TITLE
Issues related to release

### DISCUSSION THREAD

**Discussion by smallprogram** [hakimel/reveal.js#3471] [Q&A] [answered]
This question has nothing to do with the project. I just want to ask how the Changes, Fixes and Contributors are generated in the readme in your release. Is it your handwriting? I'm really curious

**Comment by t-fritsch** [comment]
Since this is not an issue, I moved this question into Discussions 🙂

**Comment by smallprogram** [comment]
> Since this is not an issue, I moved this question into Discussions 🙂 

ok

**Comment by hakimel** [comment, marked as answer]
@smallprogram I write them mostly by hand, but I use the "Generate release notes" button in the release notes editor as a foundation.

**Reply by smallprogram** [reply]
> @smallprogram I write them mostly by hand, but I use the "Generate release notes" button in the release notes editor as a foundation.

tks man

**Comment by t-fritsch** [comment]
Not sure if it can help but on some projects I use release-it https://github.com/release-it/release-it/ 

it has options to generate changelog and github/gitlab release messages : https://github.com/release-it/release-it/blob/main/docs/changelog.md
